src/styles/font.css

```diff
@@ -0,0 +1,47 @@
+@font-face {
```

Comment:
Tomorrow we will have a look on your IDE configuration: this file is not formatted